### PR TITLE
Add Minimally working capcom

### PR DESCRIPTION
**Pull request by miguelbernadi**

### CODE REVIEW

File: mcc/sgcli/cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		GroupId:       aws.String(sgid),
+		IpPermissions: []*ec2.IpPermission{perm},
+	}
+	_, err := svc.AuthorizeSecurityGroupIngress(params)
+	if err != nil {
+		log.Panic(err)
+	}
+}
+
+func RevokeIPToSecurityGRoup(svc *ec2.EC2) {
```
[the remark this answers is not in the document]
Comment (ifosch) [reply mcc/sgcli/cmd/main.go]: @miguelbernadi comment lacking here ;)

File: mcc/sgcli/cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+)
+
+var list bool
+var add bool
+var revoke bool
+var iprange string
+var port int64
+var sgid string
+
+func ListSecurityGroups(svc *ec2.EC2) {
```
[the remark this answers is not in the document]
Comment (ifosch) [reply mcc/sgcli/cmd/main.go]: and here!

File: mcc/sgcli/cmd/main.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+	for _, sg := range res.SecurityGroups {
+		fmt.Printf("* %10s %20s %s\n",
+			*sg.GroupId,
+			*sg.GroupName,
+			*sg.Description)
+	}
+}
+
+func AuthorizeIPToSecurityGroup(svc *ec2.EC2) {
```
[the remark this answers is not in the document]
Comment (ifosch) [reply mcc/sgcli/cmd/main.go]: another!

**Comment by ifosch** [comment]
@miguelbernadi +1 but fix the hound errors.
I would also suggest to look for some nicer name ;P

**Comment by miguelbernadi** [comment]
I'll fix the violations and think of a proper name.

**Comment by miguelbernadi** [comment]
Properly renamed to capcom:

The CAPCOM controller, filled by an astronaut, maintained nominal air-to-ground communications between the MCC and the orbiting spacecraft; the exception being the SURGEON or Flight Director, and only in an emergency.

**Comment by ifosch** [comment]
+1

**Pull request merged**